server/src/routes/api/event.mjs

```diff
@@ -26,17 +26,19 @@ import * as events from '../../events.mjs';
 
 // POST /api/v1/event
 // Expected body: { method: 'device.onDeviceNameChanged' result: ... }
-function sendEvent(req, res) {
+function sendEvent(req, res, sendSuccessFlag = true) {
```

Comment:
It's usually a bad idea to mix the signature of an Express route ( f(req, res) ) with any other use of parameters. Ultimately, we want this to be a regular route handler signature (no extra param) and it will probably just call sequenceManagement.sendSequence once it gets the req.body value.